### PR TITLE
First Assignment: Echo.js

### PR DESCRIPTION
**Pull request by git-ebetts**
Committed this file 3 times, each time adding progressive solutions to multi-part problem. Solutions all worked.

### CODE REVIEW

File: echo.js [hunk 1]
```diff
@@ -1,7 +1,46 @@
...
+//PART 3:
+//echo('foo', 'bar', 'baz');
+// should print
+//
+// 'foo'
+// 'bar'
+// 'baz'
+
+function echo() {
+    for (var x=0; x <= arguments.length - 1; x++)
```
Comment (afeld) [inline echo.js:42]: Minor: You can use `x < arguments.length` instead of `x <= arguments.length - 1`. Just a little shorter.

File: echo.js [hunk 2]
```diff
@@ -1,7 +1,46 @@
...
+    console.log(arguments[x]);
+}
```
Comment (afeld) [inline echo.js:44]: Good news: this version would work for all of them!

**Comment by afeld** [comment]
Looks good!

File: echo.js (hunk 2 above)
Comment (git-ebetts) [inline echo.js:44]: True! Thank you!

**Pull request closed without merging**